testing/gist/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Sascha Paunovic <azarus@posteo.net>
+# Maintainer: Sascha Paunovic <azarus@posteo.net>
+pkgname="gist"
+_pkgname="python-gist"
+pkgver=0.6.1
+pkgrel=0
+pkgdesc="gist command line interface"
+url="https://github.com/jdowner/gist"
+arch="noarch"
+license="MIT"
+depends="python3"
+makedepends="python3-dev py-setuptools"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	python3 setup.py build
+}
+
+check() {
+	cd "$builddir"
+	mkdir tests # Make checks pass
+	python3 setup.py test
+}
+
+package() {
+	cd "$builddir"
+	python3 setup.py install --prefix=/usr --root="$pkgdir"
+
+	# remove unneeded files
+	rm $pkgdir/usr/share/$pkgname/LICENSE
```

Comment:
You should quote `$pkgdir` otherwise this `rm` breaks when the absolute path contains a space.

Reply:
Agreed.